lib/page/assets/lock/LockDetailPage.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:polka_wallet/common/components/goPageBtn.dart';
+import 'package:polka_wallet/common/components/subTitle.dart';
+import 'package:polka_wallet/page/assets/lock/LockResultPage.dart';
+import 'package:polka_wallet/store/app.dart';
+import 'package:polka_wallet/utils/i18n/index.dart';
+
+class LockDetailPage extends StatefulWidget {
+  LockDetailPage(this.store);
+
+  static final String route = '/assets/lock/detail';
+  final AppStore store;
+
+  @override
+  _DetailPageState createState() => _DetailPageState(store);
+}
+
+class _DetailPageState extends State<LockDetailPage> {
+  _DetailPageState(this.store);
+
+  final AppStore store;
+
+  @override
+  Widget build(BuildContext context) {
+    var dic = I18n.of(context).assets;
+    
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(dic['lock.tokens'])
+      ),
+      body: SafeArea(
+        child: Builder(
+          builder: (BuildContext context) {
+          return ListView(
+            shrinkWrap: true, 
+            padding: const EdgeInsets.all(20),
+            children: <Widget>[
+              Text(
+                dic['transaction.message'],
+                textAlign: TextAlign.center,
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.symmetric(vertical: 10),
+                child: Text(
+                  dic['lock.transaction.instruction'],
+                  textAlign: TextAlign.center,
+                ),
+              ),
+              subTitle(dic['formula']),
+              Row(
+                children: <Widget>[
+                  formulaInput(
+                    lable: dic['lock.duration']
+                  ),
+                  Text(' , '),
+                  Expanded(
+                    child: Column(
+                      children: <Widget>[
+                        TextField(
+                          textAlign: TextAlign.center,
+                        ),
+                        Container(
+                          padding: const EdgeInsets.all(5),
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: <Widget>[
+                            Text(
+                              '${dic["signal"]},${dic["lock"]}',
+                              style: TextStyle(
+                                fontSize: 10
+                              )
+                            ),
+                            Icon(Icons.arrow_drop_down)
+                          ]),
+                        ),
+                      ],
+                    ),
+                  ),
+                  Text(' , '),
+                  formulaInput(
+                    lable: dic['amount.tokens']
+                  ),
+                ]
+              ),
```

Comment:
On this 1st "Lock Tokens" page, add an additional input field at the end of the formula (i.e. after the the input field "Amount of Tokens") with label "DHX public key (hex)" and provide a link below it with text "Guide on how to create a public key".